tests/containers/k3s_ensure_essentials_installed.pm

```diff
@@ -0,0 +1,26 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: podman docker helm
+# Summary: Ensure that helm is installed in the system if HELM_CHART is defined.
+# - check if kubectl is installed; if not, install kubectl
+# - check if helm is installed; if not, install helm
+# Maintainer: qe-c <qe-c@suse.de>
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use utils;
+use serial_terminal qw(select_serial_terminal);
+use containers::k8s qw(install_helm install_k3s);
+
+sub run {
+    select_serial_terminal;
+    return undef unless get_var('HELM_CHART');
```

Comment:
IMHO unnecessary, since this module is only loaded if the setting it set.

And if you want to add a safeguard, then it's better to `die` instead of silently "correcting" for scheduling issues.